sync_auditlog/models/log.py

```diff
@@ -0,0 +1,237 @@
+# Copyright (C) 2021 Open Source Integrators
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+from datetime import datetime
+
+try:
+    from xmlrpc import client as xmlrpclib
+except ImportError:
+    import xmlrpclib
+
+
+class AuditlogLog(models.Model):
+    _inherit = "auditlog.log"
+
+    state = fields.Selection(
+        [
+            ("logged", "Logged"),
+            ("captured", "Captured"),
+            ("Prepared", "Prepared"),
+            ("Pushed", "Pushed"),
+            ("Pulled", "Pulled"),
+            ("Processed", "Processed"),
+            ("Error", "Error"),
+            ("Failed", "Failed"),
+            ("Cancelled", "Cancelled"),
+        ],
+        string="State",
+        default="logged",
+    )
+    uuid = fields.Char("UUID")
+    timestamp = fields.Datetime("Timestamp")
+    resource_ids = fields.Char("Resource IDs")
+    raw_args_kwargs = fields.Char("Raw Args and kwargs")
+    prepared_args_kwargs = fields.Char("Prepared Args and kwargs")
```

Comment:
I was originally thinking on having args as kwargs as two separate fields.
No change needed but lets keep this in mind, in case it is helpful when implementing the "Event Apply" logic.